themes/cp-theme/red-hat-sass/variables/_typography.scss

```diff
@@ -29,21 +29,21 @@ $rh-global--FontFamily--monospace:     "Overpass Mono", Consolas, Monaco, 'Andal
 // ========================================================================
 
 // Heading sizes for Apps
-$rh-global--FontSize--heading--xxl:     rh-size-prem(32px) !default;
-$rh-global--FontSize--heading--xl:      rh-size-prem(28px) !default;
-$rh-global--FontSize--heading--lg:      rh-size-prem(24px) !default;
-$rh-global--FontSize--heading--md:      rh-size-prem(20px) !default;
-$rh-global--FontSize--heading--sm:      rh-size-prem(18px) !default;
-$rh-global--FontSize--heading--xs:      rh-size-prem(16px) !default;
-$rh-global--FontSize--heading--xxs:     rh-size-prem(14px) !default;
-
-
-// ========================================================================
-// Icons Sizes
-// ========================================================================
-
-$rh-global--icon-size--xl:              rh-size-prem(64px) !default;
-$rh-global--icon-size--lg:              rh-size-prem(48px) !default;
-$rh-global--icon-size--md:              rh-size-prem(32px) !default;
-$rh-global--icon-size--sm:              rh-size-prem(16px) !default;
-$rh-global--icon-size--xs:              rh-size-prem(14px) !default;
+$rh-global--FontSize--heading--xxl:     rh-size-prem($rh-global--font-size-root * 2)    !default;  // 32px 
```

Comment:
Should this use  `$rh-global--FontSize` instead?   $rh-global--FontSize maps to `$rh-global--font-size-root` on line 9.

Reply:
Looks good otherwise

Reply:
updated - @markcaron are you good with this, can we merge?

Reply:
I'm good with this change. We should definitely be basing the sizes for headings and icons off FontSize.